tests/forestry/grow_metsi_test.py

```diff
@@ -2,7 +2,9 @@
 from typing import cast
 from types import SimpleNamespace
 from unittest.mock import patch
+import numpy as np
 import lukefi.metsi.domain.natural_processes.grow_metsi as mg
```

Comment:
We have some inconsitency in this MetsiGrow is the repo, grow_metsi is to operation and mg is the module variable.